components/listingsPage/filters.js

```diff
@@ -199,24 +201,22 @@ const Filters = ({ t }) => {
           {mobile && mobile === true ? "+" : "-"}
         </ToggleIcon>
         <span>{t("headings.filter")}</span>
-        {process.browser ? (
-          <ClearButton onClick={(e) => clearFilters(e)}>
-            {t("headings.clear")}
-          </ClearButton>
-        ) : (
-          <a
-            href="/listings"
-            name=""
-            style={{
-              fontSize: "11px",
-              color: "#003D8E",
-              cursor: "pointer",
-              textDecoration: "none",
-            }}
-          >
-            {t("headings.clear")}
-          </a>
-        )}
+
+        <Anchor
+          onClick={(e) => clearFilters(e)}
+          textAlign="right"
+          textSize="11px"
+          width="auto"
+          href=""
+          style={{
+            textDecoration: "none",
+            background: "inherit",
+            color: "inherit",
+            outline: "none",
+          }}
```

Comment:
I think most of these styles are not necessary: I suspect we can allow this element to inherit most of its styles (except 'color') from the `<Anchor>` component: we should particularly ensure it inherits the underline on hover. I feel that the font size is a little small as well (and should preferably be defined using `rem` not `px`).

Reply:
Resolved

Reply:
👍